src/ImageSharp.Web.Providers.Azure/Providers/AzureBlobStorageImageProvider.cs

```diff
@@ -85,7 +85,7 @@ public class AzureBlobStorageImageProvider : IImageProvider
         }
 
         /// <inheritdoc/>
-        public bool ProcessWhenNoCommands { get; set; } = true;
+        public ProcessingBehavior ProcessingBehavior { get; set; } = ProcessingBehavior.All;
```

Comment:
Nearly there. Just need to drop the setters on the concrete implementations 👍

Reply:
Already done 👍  I realised I'd missed that in the initial commit so it's there in https://github.com/SixLabors/ImageSharp.Web/pull/83/commits/105a80eb9609637af9a9c36f18307356935987ae